src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentCodebase.kt

```diff
@@ -31,6 +32,12 @@ class CodyAgentCodebase(private val underlying: CodyAgentServer, private val pro
   fun onFileOpened(project: Project, file: VirtualFile) {
     application.executeOnPooledThread {
       onRepositoryNameChange(RepoUtil.findRepositoryName(project, file))
+
+      val openedFileName = file.name
+      val relativeFilePath: String? = ProjectFileUtils.getRelativePathToProjectRoot(project, file)
```

Comment:
What will be the result if someone open file from outside the workspace?

Reply:
I do not understand. what do you mean by "from outside the workspace"?

Reply:
File which is not part of the given workspace, e.g. random file from the disk which you drag to the IDE.

Reply:
![image](https://github.com/sourcegraph/jetbrains/assets/19799111/e88ad565-ab05-4112-b639-ac250bdcc581)

`my_utils.sh` is a file in `home` that does not belong to the workspace. the tooltip does not appear for it.